java/openjdk17/Portfile

```diff
@@ -21,13 +21,13 @@ distname            openjdk-${version}-ga
 use_xz              yes
 worksrcdir          jdk-${version}+${build}
 
-checksums           rmd160  d44423fb510e75d0516423573190959d7e9a284d \
-                    sha256  39a984e49a4216013e7a418fb91d46263165e5762e4ad9bd4ef95f9545e96fd6 \
-                    size    66483076
+checksums           rmd160  815a4bf32aa6f4113f71eff5ae976ab871ed385a \
+                    sha256  0ead550ff5abe15df9c2c8b3656198e6e6679186b544e1fa329436e90eb46b31 \
+                    size    66554460
 
 depends_lib         port:freetype \
                     port:libiconv
-depends_build       port:openjdk${feature}-bootstrap \
+depends_build       port:openjdk${feature}-zulu \
```

Comment:
This change has resulted in binaries which MacPorts can no longer distribute, due to the `nomirror` license of openjdk17-zulu:

https://trac.macports.org/ticket/72357